supplementary/alica_ros1/alica_tests/src/test/test_failure_handling.cpp

```diff
@@ -50,7 +51,8 @@ class AlicaFailureHandlingDisabledMultiAgentFixture : public AlicaTestMultiAgent
 protected:
     const char* getMasterPlanName() const override { return "FailureHandlingMaster"; }
     virtual int getAgentCount() const { return 2; }
-    const char* getHostName(int agentNumber) const override { return agentNumber ? "nase" : "hairy"; }
+    const char* getRoleSetName(const int agentNumber) const override { return "Roleset"; }
+    const char* getHostName(int agentNumber) const override { return agentNumber ? "hairy" : "nase"; }
```

Comment:
seems like "hairy" & "nase" have got interchanged

Reply:
I thought on changing these from the old owner of this PR, but it was adding too much diff, and the goal of this PR was just getting rid of ROS, I was eventually going to create another one to reorganize stuff and remove this from the ros folder, but I can do now if you want

Reply:
btw, the swap was intentional by hum because the entire test was built with this in mind, it fails otherwise. So I have to change the order of stuff inside the test to swap this.

Reply:
tests passed with them so seemed harmless and I didn't notice, but I put them back.